server/conf/messages

```diff
@@ -234,20 +234,13 @@ content.pleaseCreateAccount=If you sign into a city account now, your informatio
 #----------------------------------------------------------#
 
 # Address question labels - these are shown as the field label before an input box.
-label.addressLine2=Unit/Apartment Number
+label.addressLine2=Apartment, suite, etc. (optional)
 label.city=City
 label.state=State
 label.selectState=Select State
-label.street=Address line 1
+label.street=Address
```

Comment:
**suggestion (blocking):** please double check these copy changes for the applicant view of the admin fields with @mferhut